Language/Classes/Instance_Methods/Method_noSuchMethod/forwarder_A03_t01.dart

```diff
@@ -23,53 +23,44 @@
 /// For a concrete class C, a noSuchMethod forwarder is implicitly induced for
 /// each member signature which is noSuchMethod forwarded.
 ///
-/// @description Checks that it is a compile-time error if a class has an
-/// interface member `m` and doesn't have a valid concrete implementation of `m`
-/// even if it has a non-trivial `noSuchMethod`
+/// @description Checks that 'noSuchMethod' forwarder forced by privacy always
+/// throws
 /// @author sgrekhov22@gmail.com
 
-abstract mixin class A {
-  String m([String s = ""]);
-}
-
-mixin class B {
-  String m() => "B";
-}
+import "no_such_method_lib.dart";
+import "../../../../Utils/expect.dart";
 
-class C1 extends B implements A {
-//    ^^
-// [analyzer] unspecified
-// [cfe] unspecified
-  dynamic noSuchMethod(Invocation i) => "C1";
+class C extends D0 {
+  String _m();
+  dynamic noSuchMethod(Invocation i) => "C";
 }
 
-class C2 with B implements A {
-//    ^^
-// [analyzer] unspecified
-// [cfe] unspecified
-  dynamic noSuchMethod(Invocation i) => "C2";
-}
-
-mixin M on B implements A {
+mixin M on D0 {
+  String _m();
   dynamic noSuchMethod(Invocation i) => "M";
 }
 
-class MA = B with M;
-//    ^^
-// [analyzer] unspecified
-// [cfe] unspecified
+class MA = D0 with M;
 
-enum E with B implements A {
-//   ^
-// [analyzer] unspecified
-// [cfe] unspecified
+enum E with D0 {
   e1, e2;
+  String _m();
   dynamic noSuchMethod(Invocation i) => "E";
 }
 
 main() {
-  print(C1);
-  print(C2);
-  print(MA);
-  print(E);
+  Expect.equals("C", C()._m());
+  Expect.throws(() {C().test();});
```

Comment:
I think we ought to test that the `_m`-in-no_such_method_lib.dart method can be torn off, and also that it throws when invoked. So perhaps `test` should tear it off and return the function object? That function object can be called here just fine, and then we can `Expect.throws` it.